ibm_i/tests/test_e2e.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from datadog_checks.ibm_i import IbmICheck
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, aggregator, instance):
+    aggregator = dd_agent_check(instance)
+    aggregator.assert_service_check('ibm_i', IbmICheck.CRITICAL)
```

Comment:
Yes, `ibm_i.can_connect` would be the correct service check name to assert for per the `assets/service_checks.json` file.

Reply:
You can check the code to confirm the logic for when the `ibm_i.can_connect` service check gets emitted and assert accordingly, given the instance config passed to `dd_agent_check()`: https://github.com/DataDog/integrations-core/blob/master/ibm_i/datadog_checks/ibm_i/check.py

Reply:
The problem with asserting for that service check here is that, you'll more likely encounter this AttributeError exception:
https://github.com/DataDog/integrations-core/blob/master/ibm_i/datadog_checks/ibm_i/check.py#L40-L42

Reason being that we fail to reassign the query_manager which defaults to `None`, to an actual query manager object. Resulting in a `AttributeError` when we call `execute()`. 

That being said, if you want to assert for this, then either:
A) We find a way to actually get a valid query_manager
B) Modify the code so that the service check is emitted incase of a `AttributeError` exception

The latter approach is probably easier, and to me it make sense since if we can't get a query manager than technically, we can't connect to the IBM_i instance. So `ibm_i.can_connect` returning critical would make sense to me. That's my opinion atleast.

Reply:
Thanks you both! I ended up going with exactly Steven's option B!